app/streaming/streaming.go

```diff
@@ -0,0 +1,21 @@
+package streaming
+
+import (
+	"strings"
+
+	"github.com/provenance-io/provenance/app/streaming/trace"
+	"github.com/provenance-io/provenance/internal/streaming"
+)
+
+// StreamServiceInitializers contains a map of supported StreamServiceInitializer implementations
+var StreamServiceInitializers = map[string]streaming.StreamServiceInitializer{
+	"trace": trace.StreamServiceInitializer,
+}
+
+func ConfigOptions() string {
+	keys := make([]string, 0, len(StreamServiceInitializers))
+	for k := range StreamServiceInitializers {
+		keys = append(keys, k)
+	}
+	return strings.Join(keys, ",")
```

Comment:
Probably a good idea to sort the keys before returning them. The returned value might not be used anywhere to cause determinism problems, but it's probably best to be on the safe side.